scripts/test-in-lxd.sh

```diff
@@ -2,34 +2,40 @@
 set -eux
 IMAGE=$1
 SCRIPT=$2
+TESTER=subiquity-${IMAGE##*:}
 
 lxd init --auto
 
-lxc launch $IMAGE tester
-lxc config device add tester code disk source=`pwd` path=/subiquity
+if [ -z "$(lxc list -f csv -c n ^${TESTER}\$)" ]
```

Comment:
This is fine, but if it was me I'd do something like "if lxc query /1.0/containers/bl >/dev/null 2>&1; then" to avoid the regexp. But I'll merge this anyway :)